google/cloud/security/inventory/inventory_loader.py

```diff
@@ -250,25 +235,22 @@ def main(argv):
     crm_rate_limiter = RateLimiter(max_crm_calls, 100)
 
     pipelines = [
-        {'pipeline': load_projects_pipeline,
-         'status': ''},
-        {'pipeline': load_projects_iam_policies_pipeline,
-         'status': ''},
-        {'pipeline': load_org_iam_policies_pipeline,
-         'status': ''},
+        LoadOrgIamPoliciesPipeline(
```

Comment:
Or you can do a hybrid where the un-ported pipelines are still run using the older code.